service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -16,9 +16,21 @@
 
 import java.util.Arrays;
 
+/**
+ * Lagom service implementations must create one implementation of this interface and use it to bind Service
+ * implementations.
+ */
 public interface ServiceGuiceSupport extends ServiceClientGuiceSupport {
 
-    // redefines bindServiceInfo from super interface.
+    /**
+     * Creates a custom {@link ServiceInfo} for this Lagom service. This method overrides
+     * {@link ServiceClientGuiceSupport#bindServiceInfo(ServiceInfo)} with custom behavior for consume-only Lagom
+     * services.
+     * <p>
+     * The customization may be used by Lagom to add features only applicatble to Lagom services transparently.
```

Comment:
I kept it vague on purpose to keep our options open, I just wanted to point out that `ServiceGuiceSupport#bindServiceInfo` may do a bit more than `ServiceClientGuiceSupport#bindServiceInfo` because the context differs (a consume-only service vs a client).

For this reason I didn't mention `MetricsService` in `ServiceGuiceSupport#bindServiceInfo` either.